assets/ajax-cart.js.liquid

```diff
@@ -0,0 +1,627 @@
+/*============================================================================
```

Comment:
See https://github.com/cshold/Timber/commit/f9c046f83b1572ec4160815c66a5fa43cdddfda3 for the changes between ajax-cart.js and ajaxify.js.